tests/AccountTest.php

```diff
@@ -33,27 +31,25 @@ function invalid_signups_dont_proceed()
             ->see('The password field is required')
             ->see('The email field is required');
 
-        $this->assertEquals(0, User::all()->count());
+        $this->assertEquals(0, App\User::all()->count());
     }
 
     /** @test */
     function users_can_log_in()
     {
-        $user = Factory::create('user', [
-            'password' => bcrypt('super-secret'),
-        ]);
+        $user = factory(App\User::class)->create();
```

Comment:
Can we keep it where it's manually passing the password like previously so it's more obvious the connection?